src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -45,11 +45,17 @@
     <% } %>
 
     <% if(request.getSession().getAttribute("user") != null){ %>
-      <h1>New Conversation</h1>
+      <h1>Create Group Conversation</h1>
       <form action="/conversations" method="POST">
           <div class="form-group">
-            <label class="form-control-label">Title:</label>
+            <label class="form-control-label">Group Name:</label>
           <input type="text" name="conversationTitle">
+         </div>
+        <div class="form-group">
+          <label for="userLabel">Add User</label>
+          <select name = "userLabel" id= "userLabel" style="width: 300px;">
```

Comment:
We probably want the servlet to get all users that would be the options for this input.

Reply:
If you want to do that in a different PR, thats fine just leave this file off of this PR and group it with the servlet changes.

Reply:
Let's either leave this file out of this PR or at least leave a comment along the lines of "TODO(Robert): Get the list of users to populate the options for this input"

Reply:
Im doing this in another PR